response.md

```diff
@@ -63,6 +63,10 @@ Additionally, [/reverse](reverse.md) queries will have a `distance` parameter, w
 
 ## Notable features
 
+## `coordinates`
+
+At present, all results returned from Mapzen Search are points. Following the [GeoJSON specification](http://geojson.org/geojson-spec.html#positions), these coordinates are in **longitude, latitude** order.
```

Comment:
I'd lose the `At present` part, otherwise it sounds like the matter is up for debate.